lib/pbench/test/unit/server/test_endpoint_configure.py

```diff
@@ -93,6 +93,7 @@ def check_config(self, client, server_config, host, my_headers={}):
                     },
                 },
                 "endpoints": {"template": f"{uri}/endpoints", "params": {}},
+                "generate_key": {"template": f"{uri}/generate_key", "params": {}},
```

Comment:
Putting this here for lack of a better place: you need to add your new API name to the `API` ENUM in `pbench.client`, the lack of which is what broke the functional test run.